client/service/src/lib.rs

```diff
@@ -193,6 +196,12 @@ where
 		.spawn_essential_handle()
 		.spawn("cumulus-consensus", None, consensus);
 
+	// PoV Recovery is currently not supported when we connect to the
+	// relay chain via RPC, so we return early.
+	if collator_options.relay_chain_rpc_url.is_some() {
+		return Ok(())
+	}
+
```

Comment:
I don't know what the consequences of this are, but is it fine to ignore it like this? (And if so I'd be nice to write in the comment why this is okay.)

Reply:
PoV recovery was recently introduced for full-nodes: #927 .

The node should work without it. As I continue to iterate on this issue, we should be able to add it back.

Reply:
Sounds good to me! (In this case I'd suggest to simply just add a short comment or something that this is not necessary and that the node will still work without it, so that there's no confusion.)